docs/adding_packages/README.md

```diff
@@ -3,180 +3,131 @@
-
-> ⚠️ **Note**: This not a testing service, it is a binary building service for package **released**. Unit tests shouldn't be built nor run in recipes by default, see the [FAQs](../faqs.md#why-conancenter-does-not-build-and-execute-tests-in-recipes) for more. Before submitting a pull request, please ensure that it works locally for some configurations.
-
-- The CI bot will start a new build only after the author is approved. Your PR may be reviewed in the mean time, but is not guaranteed.
-- The CI system will also report with messages in the PR any error in the process, even linking to the logs to see more details and debug.
-
-The pipeline will report errors and build logs by creating a comment in the pull-request after every commit. The message will include links to the logs for inspecting.
-
-Packages generated and uploaded by this build service don't include any _user_ or _channel_ (existing references with any `@user/channel` should be considered as deprecated in favor of packages without it). Once the packages are uploaded, you will be able to install them using the reference as `name/version` (requires Conan >= 1.21): `conan install cmake/3.18.2@`.
-
-## Recipe files structure
-
-Every entry in the `recipes` folder contains all the files required by Conan to create the binaries for all the versions of one library. Those
-files don't depend on any other file in the repository (we are not using `python_requires`) and every pull-request can modify only one of those
-folders at a time.
 
-This is the canonical structure of one of these folders, where the same `conanfile.py` recipe is suitable to build all the versions of the library:
+When you are interested in contributing again, simply ask again to be included in the [issue #4](https://github.com/conan-io/conan-center-index/issues/4).
+The process will be precisely like for newcomers.
 
-```
-.
-+-- recipes
-|   +-- library_name/
-|       +-- config.yml
-|       +-- all/
-|           +-- conanfile.py
-|           +-- conandata.yml
-|           +-- patches/
-|               +-- add-missing-string-header-2.0.0.patch
-|           +-- test_package/
-|               +-- conanfile.py
-|               +-- CMakeLists.txt
-|               +-- test_package.cpp
-|           +-- test_v1_package/
-|               +-- conanfile.py
-|               +-- CMakeLists.txt
-```
+## :two: Creating a package
 
-If it becomes too complex to maintain the logic for all the versions in a single `conanfile.py`, it is possible to split the folder `all` into
-two or more folders, dedicated to different versions, each one with its own `conanfile.py` recipe. In any case, those folders should replicate the
-same structure.
+Once you've successfully built an existing recipe following [developing recipes](../developing_recipes_locally.md) tutorial.
+You are set to being.
```

Comment:
You are set to being what? I don't understand that wording

Reply:
```suggestion
You are set to being adding a new package.
```